libs/agent-scripts/cloudbroker/upgrades/2.4.0/populate_vms_from_sizeId.py

```diff
@@ -0,0 +1,21 @@
+from JumpScale import j
```

Comment:
This file should be in a subdirectory master this is how upgrader script figure out the roles see other versions for examples...